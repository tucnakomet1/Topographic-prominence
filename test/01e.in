1955 1840 1937 1837 2004 2024 1232 1660 1190 1946
