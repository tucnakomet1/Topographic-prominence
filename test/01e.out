118
97
2024
428
756
